style.css

```diff
@@ -0,0 +1,1043 @@
+  }
+
+  .nav-2-ul-2-text {
+    display: none;
+  }
+
+  .countdown {
+    display: none;
+  }
+
+  .nav-2-ul-2 button {
+    display: none;
+  }
+}
+
+@media all and (max-width: 1024px) {
+  header p {
+    padding-top: 1rem;
+    padding-bottom: 1rem;
+  }
+
+  .button-li {
+    display: none;
+  }
+
+  .hero-article div:first-child {
+    visibility: hidden;
+  }
+
+  .hero-article {
+    background: linear-gradient(to bottom right, rgba(0, 0, 0, 0.1), rgba(0, 0, 0, 0.9)), url("https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/03/bmw-car-key-apple-carkey-digital-key-car-unloack-phone-ios-code-796x448.jpg") center center scroll no-repeat;
+    background-size: cover;
+  }
+
+  .hero-article h1 {
+    font-size: 2rem;
+  }
+
+  .main-article-aside h1,
+  .main-article-aside h2 {
+    font-size: 1rem;
+  }
+
+  .grid-header {
+    width: 90%;
+  }
+
+  .latest-news-grid {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    width: 90%;
+  }
+
+  .latest-news-grid img {
+    height: 11rem;
+  }
+
+  .latest-deals-grid div img {
+    height: 11rem;
+  }
+
+  .mid-article-head-container {
+    width: 90%;
+    display: grid;
+    grid-template: 4rem 3fr / 1fr;
+    margin: 4rem auto;
+  }
+
+  .mid-article-head-title {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+  }
+
+  .mid-article-head-title ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    height: 1rem;
+  }
+
+  .last-child {
+    display: none;
+  }
+
+  .mid-article-head-content {
+    width: 100%;
+    display: grid;
+    grid-template: 1fr / repeat(3, 1fr);
+    column-gap: 1rem;
+  }
+
+  .article-content-2-container {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    width: 90%;
+  }
+
+  .article-content-2-grid {
+    margin-top: 2rem;
+  }
+}
+
+@media all and (max-width: 768px) {
+  .hamburger-menu {
+    display: block;
+  }
+
+  .footer-section-2 div {
+    font-size: 0.8rem;
+    width: 15rem;
+    text-align: center;
+    line-height: 1.3rem;
+  }
+
+  footer ul {
+    width: 20rem;
+  }
+
+  .mid-article-head-title ul {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 1rem;
+  }
+
+  .mid-article-head-title li {
+    margin-top: 1rem;
+  }
+
+  .footer-media li {
+    width: 2rem;
+    height: 2rem;
+  }
+
+  .nav-1 {
+    display: none;
+  }
+
+  .nav-2 ul:first-child {
+    display: none;
+  }
+
+  .nav-2-ul-2 {
+    width: 100%;
+    height: 3rem;
+  }
+
+  .nav-2-ul-2 li:last-child {
+    display: none;
+  }
+
+  .main-header-content {
+    height: 3rem;
+  }
+
+  .gradient-logo img {
+    height: 3rem;
+  }
+
+  .main-articles {
+    display: grid;
+    grid-template: repeat(2, 1fr) / 1fr;
+    row-gap: 0.5rem;
+    height: 15rem;
+  }
+
+  .main-articles h1 {
+    font-size: 1rem;
+  }
+
+  .main-article-head-container {
+    display: none;
+    height: 4rem;
+  }
+
+  .hero-article {
+    display: block;
+  }
```

Comment:
Same style definition of the selector on line 857. Group selectors to minimize code.

Reply:
Implemented